esp/services/ws_workunits/ws_workunitsService.cpp

```diff
@@ -3179,9 +3186,32 @@ void getWorkunitCluster(IEspContext &context, const char *wuid, SCMStringBuffer
     }
 }
 
+bool checkLogicalFileSize(IEspContext &context, const char *logicalName, __uint64 limit)
```

Comment:
I'm thinking whether the method would be more useful if change to something like 
`__unint64 getLogicalFileSize(IEspContext &context, const char *logicalName)`, and simply do a compare with the result. The logic would be more consistent with the change above also.

Reply:
Ok, will do.